pkg/apis/serving/v1beta1/trained_model_status.go

```diff
@@ -0,0 +1,13 @@
+package v1beta1
+
+import (
+	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
+)
+
+type TrainedModelStatus struct {
+	// Condition for "Deployed"
+	duckv1beta1.Status `json:",inline"`
+	// Addressable endpoint for the deployed trained model
+	// http://inferenceservice.metadata.name/v1/models/trainedmodel.metadata.name
```

Comment:
nit, can you put something like <inferenceservice.metadata.name> instead of inferenceservice.metadata.name and the same for trainedmodel. Just to show it's a substitution.

Reply:
Sure